src/components/forms/Fields/PipelineVariablesField.js

```diff
@@ -27,7 +28,7 @@ const PipelineVariablesField = ({
   supplementaryFiles,
   intl
 }) =>
-  <div>
+  <Row>
```

Comment:
Rows has to be inside <Grid fluid> (to handle margins correctly). The only exception is if you are doing top-level layout at the Page level (we do not use Grid there for the sake of better appearance).

Reply:
Inside of what? of top-level div? I do not seem to understand what you mean there...

Reply:
Sorry, I fixed MD syntax, now you can see, what I wrote :)

Reply:
Now I understand!